src/Database/Models/Tenant.php

```diff
@@ -45,6 +46,16 @@ public function getTenantKey(): int|string
         return $this->getAttribute($this->getTenantKeyName());
     }
 
+    public static function current(): self|null
```

Comment:
Unless there's a specific reason for using `self`, always use `static`.

`self` is static with limitations that make sense in certain cases.

Reply:
Yeah, I used the model name again and cs fixer changed it to `self` after I pushed. I'll change that to `static`